### ISSUE TITLE
Seed Nodes

### ISSUE THREAD

**Issue by nuclearhosting** [cryptonotefoundation/cryptonote#123]
Can I add a Seed nodes in the future and if yes, how? Its necessary to re-compile again with updated seed nodes list?

**Comment by jared201** [comment]
yes you need to recompile since the config to add new seed node is in the header file

**Comment by QuinnMallorry** [comment]
You can add a seed node from the command line, but if you want to distribute that seed node you'll need to recompile. You can recompile with updated code as much as you want, but be sure to keep abreast of changes you make that may affect consensus.

**Comment by nuclearhosting** [comment]
Perfect, thank you.

**Comment by rainmanp7** [comment]
Every seed node is added into the cryptonote config and or can be added manually.
Through the console or directly added into the config file and recompiled.
Make sure they are looking at other seed nodes of the same type.

because the updated seed nodes are in the header file... of the coin...
you can add temporary by commandline via terminal.

But always remember the coin follows the config file hard coded.
so if it changes?
say you upgrade the cofiguration file ?
then you will need to recompile and create a new Genisis
starting again with a black and compile.
Then come back with the new Genisis and compile again.
then use that.

Any changes to anything will basically make the coin conterfiet
so the geneisis and things not changing especially the configuration file that holds the seed nodes...
if you ever change that one file ?
add the seeds and you will have a new coin your making.
If you get my point :)

I hope that information helps.

**Comment by SoraKohaku** [comment]
make 1 seed soon #updates

**Comment by michaellu88** [comment]
i have try to recompile with new seed note, but the daemon still connect to old seed note. Do i need to recompile and print new genesis tx as well ? tq

**Comment by SoraKohaku** [comment]
This will flood coinbase. I think after recompile try remove peer.dat. or
something there in .(your coin) folder. And try run again.
